### ISSUE TITLE
Unique base defenses roll

### ISSUE THREAD

**Issue by worldendDominator** [Lailloken/Lailloken-UI#189]
The bar and percentage for base defenses roll on uniques counts from zero to maximum roll, which means it never goes below 85%. You can instead utilize the whole range by counting from minimum roll, the same way as BaseDefensePercentile property on trade site and in loot filter.

For example:
Steel kite shield has base armor range 146 to 167, my shield has 147. The overlay displays it as 88% = 147 / 167. With proposed change, it will display as 5% = (147-146) / (167-146).
Mirrored spiked shield has base evasion range 151 to 174, mine has 168. The overlay shows 97% = 168 / 174. With proposed change, it will be 74% = (168-151) / (174-151).

**Comment by Lailloken** [comment]
Huh, I didn't know the trade site calculates it that way. I chose the current method because I felt that taking the absolute range as the basis and displaying such low percentages introduces unnecessary negative bias towards an item when the discrepancy between min and max is rather insignificant in reality (like the 20 armor in your example, and many others).

Another factor was data-base size: holding ALT in game shows the range of that roll, but it's not included when copying the item-info into clipboard. I had to build a custom data-base for item-bases, and I tried to keep it minimal in order to also keep loading-time low.

Thanks for bringing this to my attention, I'll do some testing. It's probably better to change it to be in line with official calculations. I really appreciate the inputs/suggestions I have received from you.
